pkg/lightning/backend/local.go

```diff
@@ -1773,7 +1773,8 @@ type LocalWriter struct {
 	writer            *sstWriter
 }
 
-func (w *LocalWriter) AppendRows(ctx context.Context, tableName string, columnNames []string, ts uint64, rows Rows) error {
+// TODO: temporarily disable this async append rows before addressing the performance issue.
```

Comment:
these aren't relevant to the PR 😕.

Reply:
ok. I will put this change in another PR. 😂